Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetUITest.swift

```diff
@@ -788,75 +800,75 @@ extension PaymentSheetUITest {
         try? fillCardData(app, container: nil)
 
         app.buttons["Continue"].tap()
-        app.buttons["Checkout (Custom)"].tap()
-
-        let successText = app.staticTexts["Success!"]
-        XCTAssertTrue(successText.waitForExistence(timeout: 10.0))
-    }
-/* Disable Link test
-    func testDeferferedIntentLinkSignup_ClientSideConfirmation() throws {
-        loadPlayground(
-            app,
-            settings: [
-                "customer_mode": "new",
-                "automatic_payment_methods": "off",
-                "link": "on",
-                "init_mode": "Deferred",
-            ]
-        )
-
-        app.buttons["Checkout (Complete)"].tap()
-
-        let payWithLinkButton = app.buttons["Pay with Link"]
-        XCTAssertTrue(payWithLinkButton.waitForExistence(timeout: 10))
-        payWithLinkButton.tap()
-
-        let modal = app.otherElements["Stripe.Link.PayWithLinkViewController"]
-        XCTAssertTrue(modal.waitForExistence(timeout: 10))
-
-        let emailField = modal.textFields["Email"]
-        XCTAssertTrue(emailField.waitForExistence(timeout: 10))
-        emailField.tap()
-        emailField.typeText("mobile-payments-sdk-ci+\(UUID())@stripe.com")
-
-        let phoneField = modal.textFields["Phone"]
-        XCTAssert(phoneField.waitForExistence(timeout: 10))
-        phoneField.tap()
-        phoneField.typeText("3105551234")
-
-        // The name field is only required for non-US countries. Only fill it out if it exists.
-        let nameField = modal.textFields["Name"]
-        if nameField.exists {
-            nameField.tap()
-            nameField.typeText("Jane Done")
-        }
-
-        modal.buttons["Join Link"].tap()
-
-        // Because we are presenting view controllers with `modalPresentationStyle = .overFullScreen`,
-        // there are currently 2 card forms on screen. Specifying a container helps the `fillCardData()`
-        // method operate on the correct card form.
-        try fillCardData(app, container: modal)
-
-        // Pay!
-        let payButton = modal.buttons["Pay $50.99"]
-        expectation(for: NSPredicate(format: "enabled == true"), evaluatedWith: payButton, handler: nil)
-        waitForExpectations(timeout: 10, handler: nil)
-        payButton.tap()
+        app.buttons["Confirm"].tap()
 
         let successText = app.staticTexts["Success!"]
         XCTAssertTrue(successText.waitForExistence(timeout: 10.0))
     }
-*/
+    /* Disable Link test
+     func testDeferferedIntentLinkSignup_ClientSideConfirmation() throws {
+     loadPlayground(
+     app,
+     settings: [
+     "customer_mode": "new",
+     "automatic_payment_methods": "off",
+     "link": "on",
+     "init_mode": "Deferred",
```

Comment:
I think we should maybe just delete tests rather than comment them out -- but if we want to keep them commented out, maybe we should port them to use the new settings struct

Reply:
Agreed that we should delete them, but maybe not in this PR. Replacing the living tests was a lot of manual effort, so I don't really want to do it for a bunch of commented out tests too